eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Snap.hs

```diff
@@ -52,18 +54,19 @@ instance (UsesTxOut era, UsesValue era) => STS (SNAP era) where
   transitionRules = [snapTransition]
 
 snapTransition ::
-  ( UsesValue era,
-    HasField "address" (Core.TxOut era) (Addr (Crypto era))
-  ) =>
+  forall era.
   TransitionRule (SNAP era)
 snapTransition = do
   TRC (lstate, s, _) <- judgmentContext
 
-  let LedgerState (UTxOState utxo _ fees _) (DPState dstate pstate) = lstate
-      stake = stakeDistr utxo dstate pstate
+  let LedgerState (UTxOState _utxo _ fees _ incStake) (DPState dstate pstate) = lstate
+      -- stakeSnap = stakeDistr @era utxo dstate pstate
```

Comment:
we can delete this comment. bye bye :wave: `stakeSnap` :tada:

Reply:
I was thinking we might leave it as a bit of history, if we ever wanted to back this out, this tells us where we would have to make the one big important change. Think about it.

Reply:
How about it we just pretty it up a bit:

```
-- | The stake distribution was previously computed as in the spec:
--
-- @
--  stakeDistr @era utxo dstate pstate
-- @
--
-- but is now computed incrementally.
```
```

Reply:
Done